spec/row-chart-spec.js

```diff
@@ -352,11 +352,30 @@ describe('dc.rowChart', function () {
                     });
                 });
             });
+
+            if (xAxisTicks) {
+                describe('with elasticX', function () {
+                    beforeEach(function () {
+                        chart.elasticX(true)
+                            .xAxis().ticks(3);
+
+                        chart.render();
+                    });
+
+                    it('should generate x axis domain dynamically', function () {
+                        var nthText = function (n) { return d3.select(chart.selectAll('g.axis .tick text')[0][n]); };
+
+                        for (let i = 0; i < xAxisTicks.length; i++) {
```

Comment:
Wow, that's interesting (and scary)! Sloppy to let a `let` (sic.) slip in there; I am truly ES6 damaged.

I don't remember if I did something out of the ordinary, and I have the code and dev env set up on another computer I can't access right now... :o